modules/Core/pages/home.php

```diff
@@ -32,10 +32,12 @@
 }
 
 $home_type = Util::getSetting('home_type');
+$forum_enabled = Util::isModuleEnabled('Forum');
 
 $smarty->assign('HOME_TYPE', $home_type);
+$smarty->assign('FORUM_ENABLED', $forum_enabled);
 
-if ($home_type === 'news') {
+if ($home_type === 'news' && $forum_enabled) {
```

Comment:
Can't use forum_enabled here, The front_page_modules  is for any modules to use, so its also wrong that you renamed it to news

Like my server list module using the front_page_modules

Reply:
the entire frontpage stuff need a rework anyways